edxval/serializers.py

```diff
@@ -99,13 +99,14 @@ def to_internal_value(self, data):
         """
         Convert data into CourseVideo instance and image filename tuple.
         """
-        if isinstance(data, basestring):
-            course_id, image = data, None
-        elif  isinstance(data, dict):
-            (course_id, image), = data.items()
+        course_video = course_id = data
```

Comment:
Note to reviewers: These changes in `to_internal_value ` are required because we are getting `UnboundLocalError: local variable 'course_id' referenced before assignment` error when data is evaluated `False`, flow is not going in any of the `if` statements.